addon/styles/addon.scss

```diff
@@ -3,9 +3,9 @@
 .frost-object-browser {
   display: flex;
   flex-direction: column;
-  padding: 0 30px;
 
   .frost-info-bar {
+    padding: 0 30px;
```

Comment:
Info bar styles should be changed globally as part of https://github.com/ciena-frost/ember-frost-info-bar/blob/master/addon/styles/addon.scss